app/routes/products.$categorySlug/route.tsx

```diff
@@ -40,20 +41,28 @@ export const handle = {
 export default function ProductsPage() {
     const {
         category,
-        categoryProductSlice: initialResultsFromLoader,
+        categoryProducts: productsFromLoader,
         allCategories,
         productPriceBounds,
     } = useLoaderData<typeof loader>();
 
     const { appliedFilters, someFiltersApplied, clearFilters, clearAllFilters } =
         useAppliedProductFilters();
     const { sorting } = useProductSorting();
-    const { products, totalProducts, loadMoreProducts, isLoadingMoreProducts } = useProductResults({
-        categorySlug: category.slug!,
-        filters: appliedFilters,
-        sorting,
-        initialResultsFromLoader,
-    });
+    const resultsFromLoader = useMemo<ProductsPageResults>(
+        () => ({
+            products: productsFromLoader.items,
+            totalProducts: productsFromLoader.totalCount,
+        }),
+        [productsFromLoader],
+    );
```

Comment:
Why not just pass it directly without renaming properties?

Reply:
I thought `products` is a little bit better name then `items`, but both are fine for me, fixed